src/hapi_schema/db_conflict_event.py

```diff
@@ -77,6 +80,29 @@ class DBConflictEvent(Base):
         DBAdmin2.name.label("admin2_name"),
         DBAdmin2.is_unspecified.label("admin2_is_unspecified"),
         DBAdmin2.admin1_ref.label("admin1_ref"),
+        case(
```

Comment:
This code appears repeatedly, I've wondering if there is a way of factoring it out

Reply:
I have done so